src/internal.c

```diff
@@ -1539,25 +1539,26 @@ int wolfSSL_dtls_import_internal(WOLFSSL* ssl, byte* buf, word32 sz)
     idx += ret;
 
     SetKeysSide(ssl, ENCRYPT_AND_DECRYPT_SIDE);
+    /* make sure is a valid suite used */
+    if (wolfSSL_get_cipher(ssl) == NULL) {
+        WOLFSSL_MSG("Can not match cipher suite imported");
+        return MATCH_SUITE_ERROR;
+    }
 
+#ifndef WOLFSSL_AEAD_ONLY
```

Comment:
Why are these 2 checks an error in AEAD only mode?

Reply:
In `WOLFSSL_AEAD_ONLY` the following are not defined:
- `ssl->hmac`
- `TLS_hmac`
- `stream`
These two checks don't make sense in the context of `WOLFSSL_AEAD_ONLY` because authentication is built into the AEAD ciphers so the `ssl->hmac` is never used and there is no need to check if it is a `stream` or `wolfssl_cipher_null` cipher since they are not compiled in.

Reply:
Thanks.